bundles/action/org.openhab.action.pushover/src/main/java/org/openhab/action/pushover/internal/Pushover.java

```diff
@@ -266,6 +274,13 @@ public static boolean pushover(@ParamDoc(name = "apiKey", text = "Your applicati
                     logger.error("The url is greater than " + API_MAX_URL_LENGTH + " characters.");
                     return false;
                 }
+            } else if (StringUtils.isEmpty(url) && !StringUtils.isEmpty(defaultUrl)) {
```

Comment:
The part `StringUtils.isEmpty(url) && ` is unneeded and would read better if it were removed.